### DISCUSSION TITLE
https://gethue.com/ can not be visited

### DISCUSSION THREAD

**Discussion by ChenRussell** [cloudera/hue#3913] [Questions] [answered]
Why https://gethue.com/ can not be visited? This is so sad

**Comment by agl29** [comment, marked as answer]
Apologies for the inconvenience. Could you please check again? It seems there was an issue, but we’ve resolved it, and it should now be working as expected.
Thank you, @ChenRussell for reporting this!

**Reply by ChenRussell** [reply]
Thanks, it's working now.

> Apologies for the inconvenience. Could you please check again? It seems there was an issue, but we’ve resolved it, and it should now be working as expected. Thank you, @ChenRussell for reporting this!
